phoenicis-javafx/src/main/java/org/phoenicis/javafx/controller/apps/AppsController.java

```diff
@@ -19,67 +19,59 @@
 package org.phoenicis.javafx.controller.apps;
 
 import javafx.application.Platform;
-import org.phoenicis.javafx.views.mainwindow.apps.ApplicationsView;
+import org.phoenicis.javafx.components.application.control.ApplicationView;
+import org.phoenicis.javafx.dialogs.ErrorDialog;
 import org.phoenicis.repository.RepositoryManager;
 import org.phoenicis.repository.dto.CategoryDTO;
 import org.phoenicis.repository.dto.RepositoryDTO;
 
 import java.util.List;
-import java.util.Optional;
 
 import static org.phoenicis.configuration.localisation.Localisation.tr;
 
 public class AppsController {
-    private final ApplicationsView view;
-    private final RepositoryManager repositoryManager;
-
-    private boolean firstViewSelection = true;
+    private final ApplicationView view;
 
     private Runnable onAppLoaded = () -> {
     };
 
-    public AppsController(ApplicationsView view, RepositoryManager repositoryManager) {
+    public AppsController(ApplicationView view, RepositoryManager repositoryManager) {
```

Comment:
Bit inconsistent that it's `AppsController` but `ApplicationView`. We should use:
- `ApplicationsController` and `ApplicationsView`
- `AppsController` and `AppsView`

Reply:
In this PR I want to minimize the number of contained changes which have no direct relation to the refactoring of the `ApplicationsView`. 

Long-term I plan to remove the `AppsController` class. In addition I want to remove the injection mechanism for our main view instances via `@Bean` methods. Instead the creation of the main view instances should be performed inside the `MainWindow` class.

Reply:
Ok.